tracer/test/Datadog.Trace.Tests/Telemetry/TelemetryControllerTests.cs

```diff
@@ -134,6 +135,7 @@ public async Task TelemetrySendsHeartbeatsIndependentlyOfData()
         [Fact]
         public async Task TelemetryControllerAddsAllAssembliesToCollector()
         {
+            JTokenAssertions.Initialize();
```

Comment:
I don't think I've used a `[ModuleInitializer]` before, but reading through the docs for it, is this method call needed? It sounds like it would be called automatically?